src/reporting/private/dashboard.ts

```diff
@@ -91,10 +106,21 @@ export const analysisPermissionActions = [
   'quicksight:DescribeAnalysisPermissions',
 ];
 
-export const dashboardPermissionActions = [
+export const dashboardReaderPermissionActions = [
+  'quicksight:DescribeDashboard',
+  'quicksight:ListDashboardVersions',
+  'quicksight:QueryDashboard',
+];
+
+export const dashboardAdminPermissionActions = [
   'quicksight:DescribeDashboard',
```

Comment:
use `...dashboardReaderPermissionActions` to reuse the reader's permission set

Reply:
fixed